src/silverlabnwb/nwb_file.py

```diff
@@ -693,21 +738,22 @@ def read_functional_data(self, folder_path):
                                         [len(all_rois), cycles_per_trial]))[::-1]
         self._write_roi_data(all_rois, len(trials), cycles_per_trial, ch_data_shape, folder_path)
 
-    def add_custom_silverlab_data(self):
+    def add_custom_silverlab_data(self, include_opto=True):
         metadata_class = get_class('SilverLabMetaData', 'silverlab_extended_schema')
```

Comment:
Could be split into `add_custom_silverlab_metadata` and `add_custom_silverlab_optophysiology` instead. This is probably nitpicking.

Reply:
It could, yes. I left it that way so we don't change the existing code more. I'm leaning towards keeping it like this because it's one step conceptually, at least in my mind.